Cargo.lock

```diff
@@ -2892,28 +2892,28 @@ dependencies = [
  "bytes 0.5.6",
  "futures 0.3.5",
  "lazy_static",
- "libp2p-core",
+ "libp2p-core 0.20.1",
```

Comment:
You've probably forgotten to update a `Cargo.toml` somewhere. It seems that we're now using two different versions of libp2p at the same time.

Reply:
Good catch. I did a bad merge in 22cb019d7c3ee1fd4b23bc87b9959edb50714b86 not updating the libp2p version introduced in https://github.com/paritytech/substrate/pull/4954.